packages/common/src/services/WatchHistoryService.ts

```diff
@@ -1,20 +1,23 @@
 import { injectable } from 'inversify';
 
-import * as persist from '../utils/persist';
 import type { PlaylistItem } from '../../types/playlist';
 import type { SerializedWatchHistoryItem, WatchHistoryItem } from '../../types/watchHistory';
 import type { Customer } from '../../types/account';
 
 import ApiService from './ApiService';
+import StorageService from './StorageService';
 
 @injectable()
 export default class WatchHistoryService {
-  private PERSIST_KEY_WATCH_HISTORY = `history${window.configId ? `-${window.configId}` : ''}`;
+  private PERSIST_KEY_WATCH_HISTORY = 'history';
```

Comment:
This doesn't this break all the currently stored watch histories?

Reply:
`window.configId` is not being used anymore. This variable is never set and only consumed here. I removed it because of the `window` usage.